index.js

```diff
@@ -164,124 +157,136 @@ function onRequestResponse (options, error, response, body) {
     return callback(null, response, body);
   }
 
+  // Decompress brotli compressed responses
+  if (/\bbr\b/i.test('' + response.caseless.get('content-encoding'))) {
+    if (!brotli.isAvailable) {
+      const cause = 'Received a Brotli compressed response. Please install brotli';
```

Comment:
Just wondering, when this brotli is used? I hear about it for the first in my life

Reply:
Think gzip. It's just another compression format. If a server sends a Brotli encoded response, it should set the content-encoding header to `br`. Likewise if a server sends a gzip encoded response, it should set the same header to `gzip`. All major browsers support this compression format and report it as an accepted encoding via the `accept-encoding` header. If you want to know more, I would suggest googling Brotli.

Reply:
Interesting, thank you for the explanation. Of course I come across gzip frequently, but never ever heard about Brotli